scripts/arm32_ci_script.sh

```diff
@@ -203,9 +264,17 @@ do
             exit_with_error "--buildConfig can be only Debug or Release" true
         fi
         ;;
+    --arm)
+        __ARMRootfsImageBase="rootfs-u1404.ext4"
```

Comment:
Sounds good - let me know your thoughts on testing once you have ideas on how you want to move ahead with it.